src/util/dstring.h

```diff
@@ -13,6 +13,9 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "string_container.h"
 
+class serializert;
+
```

Comment:
Are you making up new coding standards just to make things less structured/readable?